### PR TITLE
Console landing page applet system.

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-12681

This replaces the hardcoded landing page content with simple renderers for now with hard-coded schemas. There are no visual changes.

API content and schema clean up will be added in a follow-up.

### CODE REVIEW

File: src/components/app-content-renderer/config-try-learn-tile.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  cloudSecurity: CloudSecurityIcon,
+  ansible: IconAnsible,
+};
+
+console.log({ iconMapper });
+
+const NoIcon = () => <div>No icon</div>;
+const TileItem = ({ icon, title, link: { href, title: linkTitle } = {} }) => {
+  const Icon = iconMapper[icon] || NoIcon;
+  console.log(Icon, icon);
```
Comment (karelhala) [inline src/components/app-content-renderer/config-try-learn-tile.js:35]: Extrta `console.log`
```suggestion
```

File: src/components/app-content-renderer/config-try-learn-tile.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+const iconMapper = {
+  connected: ConnectedIcon,
+  insights: IconInsights,
+  builderImage: BuilderImageIcon,
+  couldTenant: CloudTenantIcon,
+  cloudSecurity: CloudSecurityIcon,
+  ansible: IconAnsible,
+};
+
+console.log({ iconMapper });
```
Comment (karelhala) [inline src/components/app-content-renderer/config-try-learn-tile.js:30]: Extra `console.log`
```suggestion
```

File: src/components/app-content-renderer/config-try-learn-tile.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            </a>
+          </Text>
+        </TextContent>
+      </SplitItem>
+    </Split>
+  );
+};
+
+TileItem.propTypes = {
+  icon: PropTypes.oneOf(['connected', 'insights']).isRequired,
```
Comment (karelhala) [inline src/components/app-content-renderer/config-try-learn-tile.js:61]: Shouldn't this be all keys from `iconMapper`
```suggestion
  icon: PropTypes.oneOf(Object.keys(iconMapper)).isRequired,
```

File: src/components/app-content-renderer/config-try-learn-tile.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+  title: PropTypes.string.isRequired,
+  link: PropTypes.shape({
+    href: PropTypes.string.isRequired,
+    title: PropTypes.string.isRequired,
+  }).isRequired,
+};
+
+const ConfigTryLearnTile = ({ title, items }) => {
+  console.log(items);
```
Comment (karelhala) [inline src/components/app-content-renderer/config-try-learn-tile.js:70]: Extra `console.log`
```suggestion
```

File: src/components/app-content-renderer/config-try-learn-tile.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+  return (
```
Comment (karelhala) [inline src/components/app-content-renderer/config-try-learn-tile.js:71]: Extra `return` statement.

File: src/components/app-content-renderer/config-try-learn-tile.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+      ))}
+    </Flex>
+  );
+};
+
+ConfigTryLearnTile.propTypes = {
+  title: PropTypes.string.isRequired,
+  items: PropTypes.arrayOf(
+    PropTypes.shape({
+      icon: PropTypes.oneOf(['connected', 'insights']).isRequired,
```
Comment (karelhala) [inline src/components/app-content-renderer/config-try-learn-tile.js:90]: Same as above, I'd use all keys of `iconMapper`
```suggestion
      icon: PropTypes.oneOf(Object.keys(iconMapper)).isRequired,
```

**Review by rvsia — APPROVED** [review]

File: src/components/app-content-renderer/recommendation-renderer.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import RecommendationTile from './recommendation-tile';
+const recommendationRenderer = (recommendations) => {
+  return recommendations.map((rec) => (
```
Comment (karelhala) [inline src/components/app-content-renderer/recommendation-renderer.js:4]: Extra `return` statement

File: src/components/app-content-renderer/recommendation-tile.js [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+  description,
+  action,
+}) => {
+  const GroupIcon = groupIconMapper[icon] || NoIcon;
+  console.log({
+    icon,
+    action,
+    description,
+    component,
+  });
```
Comment (karelhala) [inline src/components/app-content-renderer/recommendation-tile.js:41]: Extra `console.log`

File: src/layout/landingPage/Footer.js [hunk 9]
```diff
@@ -1,195 +1,82 @@
...
-  CloudTenantIcon,
-  ConnectedIcon,
-  ExternalLinkAltIcon,
-} from '@patternfly/react-icons';
+import { Flex } from '@patternfly/react-core';
+import configTryLearnRenderer from '../../components/app-content-renderer/config-try-learn-renderer';
 
-import IconInsights from '../../components/icon-insights';
-import IconAnsible from '../../components/icon-ansible';
+const mockData = [
```
Comment (karelhala) [inline src/layout/landingPage/Footer.js:5]: No need to use mockData, you can pass these values directly to https://github.com/RedHatInsights/landing-page-frontend/blob/master/src/utils/config.json#L84

**Pull request merged**